simulators/main/shiftplan.py

```diff
@@ -14,16 +14,17 @@
 class Shiftplan:
     polling_interval = one_day
```

Comment:
I know it is n not related but I'd assume we dont need that variable anymore b/c we're not polling the shiftplan anymore ... please also check for usages of this variable.